doc/dbus/org.opensuse.Agama1.Network.Connections.doc.xml

```diff
@@ -41,13 +35,15 @@
       <arg name="id" type="s" direction="in"/>
     </method>
     <!--
-     Apply:
      Applies the network configuration.
 
      It includes adding, updating and removing connections as needed.
      -->
     <method name="Apply">
     </method>
+    <signal name="ConnectionAdded">
+      <arg name="id" type="s"/>
+      <arg name="path" type="s"/>
```

Comment:
The same *path* kind as above I guess, what is it? Should it be `type="o"`?

Reply:
No, this one is a D-Bus path, not related to the kernel path of the device. But, yes, it should be an `o`.